### PR TITLE
Handlebar helper lorem

### PR DESCRIPTION
**Pull request by mikesealey**
## Description
This PR is combined with [this release](https://github.com/Budibase/handlebars-helpers/releases/tag/v0.13.2) in the Handlebar-Helpers repo to add a Lorem Ipsum generator to the manifest. 

`{{ lorem }}` will return "Lorem ipsum" 
`{{ lorem n }}` will return _n-characters_ of lorem ipsum, up to a maximum of 8032 characters.
eg `{{ lorem n }} => Lorem ipsum dolor sit amet, consectetur adipiscing`

## Screenshots
image

## Launchcontrol
Adds Lorem ipsum generator to handlebar-helpers

### CODE REVIEW

**Review by mike12345567 — APPROVED** [review]
LGTM - great improvement!

File: packages/server/src/jsRunner/tests/jsRunner.spec.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ describe("jsRunner (using isolated-vm)", () => {
   describe("helpers", () => {
     runJsHelpersTests({
       funcWrap: (func: any) => config.doInContext(config.getAppId(), func),
-      testsToSkip: ["random", "uuid"],
+      testsToSkip: ["random", "uuid", "lorem"],
```
Comment (adrinr) [inline packages/server/src/jsRunner/tests/jsRunner.spec.ts:53]: Why this? This should work automatically based on the test example

**Review by adrinr — CHANGES_REQUESTED** [review]

File: packages/server/src/jsRunner/tests/jsRunner.spec.ts (hunk 1 above)
Comment (mike12345567) [reply packages/server/src/jsRunner/tests/jsRunner.spec.ts:53]: Sorry - by bad - I left this in after some fiddling!

**Review by mike12345567 — APPROVED** [review]
changes addressed.

**Review by adrinr — APPROVED** [review]
🚀

**Pull request merged**